_posts/2018-04-19-working-group-invitation.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Extending an Invitation to the Working Group Contributors"
+authors:
+  - hoverbear
+---
+
+If you're a RustFest regular you may recall that last year our friends at 1aim offered conference tickets to some participants in the Libz Blitz. This year they reached out to us regarding supporting more of our community getting to RustFest. But the big question was "How!?"
+
+You may recall as part of [the 2018 roadmap (RFC 1728)](https://github.com/rust-lang/rfcs/pull/2314) and other community efforts multiple working groups have been formed around different topics. Yann, 1aim's co-founder, reached out to these working groups and spoke with them about how to best support the teams and support contributors.
+
+## What is a Working Group?
+
+[Working groups](https://internals.rust-lang.org/t/announcing-the-2018-domain-working-groups/6737) are assemblies of contributors. Working Groups work directly with the Core Team and focus on the imporving the end-to-end user experience of using Rust in their respecitve domain. That work can involve the language, compiler, libraries, tools, documentation, discoverability, and more! The overall effort is organized by Aaron Turon (`aturon`).
+
+* Networking services, lead by `withoutboats` and `cramertj`
+* [WebAssembly](https://internals.rust-lang.org/t/come-join-the-rust-and-webassembly-working-group/6845), lead by `fitzgen`
+* [CLI apps](https://internals.rust-lang.org/t/announcing-the-cli-working-group/6872), lead by `killercup`
+* [Embedded devices](https://internals.rust-lang.org/t/announcing-the-embedded-devices-working-group/6839), lead by `japaric`
```

Comment:
Is this a complete list? Do we need to make it more clear if it is not?

Reply:
It's not. But these are the teams I understand to be participating.